components/picker/stories/picker.test.js

```diff
@@ -9,24 +9,88 @@ export const PickerGroup = Variants({
 	},
 	testData: [
 		{
-			testHeading: "Default",
+			testHeading: "Default, with placeholder",
 		},
 		{
-			testHeading: "Explicit",
-			variant: "explicit",
+			testHeading: "Default, with value and text overflow",
+			currentValue: "The selected value of the picker, with long text the triggers the overflow behavior with ellipsis",
 		},
 		{
-			testHeading: "Button",
-			variant: "button",
+			testHeading: "Quiet",
+			isQuiet: true,
+		},
+		{
+			testHeading: "Side label",
+			labelPosition: "side",
+		},
+		{
+			testHeading: "Side label, alignment with switch",
+			labelPosition: "side",
+			withSwitch: true,
+		},
+		{
+			testHeading: "With thumbnail icon",
+			showWorkflowIcon: true,
 		},
 	],
 	stateData: [
+		{
+			testHeading: "Hovered",
+			isHovered: true,
+		},
+		{
+			testHeading: "Active",
+			isActive: true,
+		},
+		{
+			testHeading: "Keyboard focused",
+			isKeyboardFocused: true,
+		},
+		{
+			testHeading: "Invalid",
+			isInvalid: true,
+		},
+		{
+			testHeading: "Invalid + hovered",
+			isInvalid: true,
+			isHovered: true,
+		},
+		{
+			testHeading: "Loading",
+			isLoading: true,
+		},
+		{
+			testHeading: "Loading + hovered",
+			isHovered: true,
```

Comment:
Correct me if I'm wrong, but does this need the `isLoading: true`?

Reply:
Yes! Good catch. Updated.